### ISSUE TITLE
Duplicate items, unneccessary items

### ISSUE THREAD

**Issue by userz** [vgstation-coders/vgstation13#207]
As of current there are many unneccessary duplicate items that might clog up the itembase aimlessly, some of them: many variants of brown shoes (captain, hop...), many variants of black gloves (hos, ce). If this ever goes in, remember not to miss Heads of Staff that might possess these items 'on start' (although they probably don't). Also found some 'fluff' items in /under/ (jumpsuits) tab. Energy guns (taser gun with blue ammunition) from NanoTrasen Navy Officer-alikes (Select Equipment menu) are also useless, they deal no damage, don't stun or anything.

tl;dr - lots of useless duplicates going around, just check the /under/ or /gun/ tab. You might also like to type 'combat' in the in-game spawn item menu or check for 'shoes' and 'gloves'.
- Cut out SWAT/Combat gear part of text, they do give different bonuses!
